not-to-release/sources/email/enronsent32_01.xml.conllu

```diff
@@ -334,10 +334,10 @@
 19	obligations	obligation	NOUN	NNS	Number=Plur	15	nmod	15:nmod:of	_
 20	under	under	ADP	IN	_	28	case	28:case	_
 21	the	the	DET	DT	Definite=Def|PronType=Art	28	det	28:det	_
-22	Indian	Indian	PROPN	NNP	Number=Sing	24	compound	24:compound	_
+22	Indian	Indian	ADJ	NNP	Degree=Pos	24	amod	24:amod	_
```

Comment:
This tree looks messed up?

Reply:
I think Roman numerals in non-flat names are treated as heads right now (e.g. "World War II"). Not ideal but it's a separate issue.

Reply:
#167